src/pages/404/index.tsx

```diff
@@ -1,13 +1,23 @@
 import Link from 'next/link';
 import Layout from '@/components/Layout';
-import styles from '@/components/Layout/Layout.module.scss';
+import styles from '@/pages/404/404.module.scss';
+import Image from 'next/image';
 
 export default function PageNotFound() {
     return (
         <Layout>
             <Link href="/" passHref>
-                <div className={styles.header}>
-                    <h2 className={styles.link}>404 - Page Not Found</h2>
+                <div className={styles.notFound}>
+                    <Image
+                        src="/not-found.jpg"
+                        alt="404 - Page Not Found"
+                        width={500}
+                        height={500}
+                        className={styles.notFoundImage}
+                    />
+                    <h2 className={styles.notFoundText}>
+                        404 - Page Not Found
+                    </h2>
                 </div>
             </Link>
```

Comment:
The link doesn't indicate in the UI that it will go to homepage. Can this be changed? A separate link for go to homepage maybe.

Reply:
Ok